types/src/main.ts

```diff
@@ -1860,6 +1860,11 @@ export class DocumentSymbol {
 	 */
 	range: Range;
 
+	/**
+	 * Uri of the document.
+	 */
+	uri?: string
```

Comment:
See: [Added an optional `uri` property to the `DocumentSymbol`.](https://github.com/Microsoft/vscode-languageserver-node/pull/425)